Some graphics for the Ember Core Concepts section

alexUXUI opened the pull request:
I have .svg .ai files as well. Would like to get approval before I submit. Note: These files are intentionally huge. I can scale down for production - but these are quality.

![ember-app-core-concepts](https://cloud.githubusercontent.com/assets/9002289/11152666/2b0bdbf4-89f1-11e5-9275-d081cf2c76c4.gif)
![ember-app-core-concepts](https://cloud.githubusercontent.com/assets/9002289/11152669/2d01ece6-89f1-11e5-9b6c-8f063b85c411.jpg)

locks committed:
Imports thoughtbot redesign styles

kellywerks committed:
update core concepts with diagram

kellywerks committed:
add models section for tutorial

kellywerks committed:
add ember-data with firebase to getting started

kellywerks committed:
add application setup and structure file to getting-started

kellywerks committed:
add first component section for tutorial (hide/show image)

kellywerks committed:
add route and templates file for tutorial with images

michaelrkn committed:
flesh out explanation of a 'hook'

michaelrkn committed:
touch up explanations in routes/templates tutorial

locks committed:
Formats CLI input to match ember-cli

Commit:
Add a missing backtick to tutorial component page

bmichelsen committed:
Validate HTML files during build

acorncom committed:
Fixed page layout so the content and sidebar work smoother

acorncom committed:
Cleaned up the code highlighting box to fix the thoughbot design

acorncom committed:
Migrated over some missing thoughtbox sidebar styles

acorncom committed:
Removed underlining that occured on our sidebar navigation items

locks committed:
Tweaks code sample styling

acorncom committed:
Fixing overall wrapper setup

acorncom committed:
Fixed the way our previous and back links looked

locks committed:
Make the TOC slide snappier

acorncom committed:
Setup initial header using the original design

locks committed:
move pages to tutorial

josemarluedke committed:
Change header color to ember orange

josemarluedke committed:
Remove github link from header

josemarluedke committed:
Tweak line numbers background to match code block

josemarluedke committed:
Cleanup unnecessary styles

josemarluedke committed:
Remove unused images

josemarluedke committed:
Remove unused fonts

josemarluedke committed:
Refactor styles and merge old styles with new

acorncom committed:
Shifted toward the thoughtbot header style

acorncom committed:
Fix for the header.svg file

acorncom committed:
Tweaked heading spacing

acorncom committed:
Handles various small bugs around spacing and lists

Commit:
Minorl language changes

A few minor language changes to the first half of the page.

michaelrkn committed:
Merge pull request #971 from BillyZac/patch-1

Minor language changes

alexUXUI committed:
Updated first Ember doc

Took out 'you' on line 6 and changed sentence structure on line 7. (clarity)

baconsaur committed:
Updated diagrams to fit with overall style

acorncom committed:
First draft of the Ember-CLI-Mirage changes for the data pages

We're getting errors here:

Error while processing route: index Adapter operation failed Error: Adapter operation failed

Mirage and Ember aren't matching up yet and we have to handle
JSON-API upgrade issues as well ...

michaelrkn committed:
Merge pull request #972 from alexG53090/patch-1

Updated first Ember doc

michaelrkn committed:
Merge pull request #973 from baconsaur/master

Updated diagrams to fit with overall style

acorncom commented:
@alexG53090 Could you open an issue over on https://github.com/emberjs/guides/issues with those pictures?  Using an issue is a good way to start a discussion without actually sending over files or any changes :+1: Thanks for the work!

Commit:
Spelling corrections

locks committed:
Merge pull request #980 from mwils/master

Spelling corrections

locks committed:
Tweaks code sample

rwjblue committed:
Update Factory Lookup section.

This section is talking about how to get a singleton instance from the container, and not how to grab the factory itself (so that you might later be able to instantiate it yourself).  

Before this change, based on the title alone I would have expected it to discuss `appInstance._lookupFactory` (or `container.lookupFactory`), which is private.

After this change, it seems clearer that we are looking up an instance not the class itself.

michaelrkn committed:
Merge pull request #981 from emberjs/rwjblue-patch-1

Update Factory Lookup section.

tomdale committed:
Life-cycle -> Lifecycle

locks committed:
Merge pull request #985 from tomdale/lifecycle

Life-cycle -> Lifecycle

tomdale committed:
Tweaks to component lifecycle guide

- General language/clarity cleanup
- Move from numeric IDs to semantic IDs (this is more friendly to
  refactorings in the future)
- Remove mention of `_super()` - `didInsertElement` should not require
  this on Ember.Component subclasses. Subclasses of other components
  should follow the normal rules of `_super()` invocation

alexUXUI committed:
update core concepts img

Glennvd committed:
Added `type` to acceptable input attributes

locks committed:
Merge pull request #989 from Glennvd/master

Added `type` to acceptable input attributes

vitch committed:
Switching to use the new `moduleForAcceptance` helper

The latest `ember-cli` (`1.13.12`) provides a `moduleForAcceptance` helper. This PR updates the guides to use it.

michaelrkn committed:
Merge pull request #990 from vitch/patch-1

Switching to use the new `moduleForAcceptance` helper

Commit:
Fix quotes in customizing element example

michaelrkn committed:
Merge pull request #991 from knownasilya/patch-1

Fix quotes in customizing element example

locks committed:
Rewords PhantomJS optional dependency install instructions

Commit:
Added spellchecking during build - closes #948
Using gem middleman-spellcheck
Added allowed wordlist
Added info to README

trcarden committed:
Removing need/have extra word

Removing need and leaving have per @michaelrkn recommendation

michaelrkn committed:
Merge pull request #999 from trcarden/patch-2

Removing need/have extra word

michaelrkn committed:
Merge pull request #994 from emberjs/locks-patch-1

Remove PhantomJS install instructions

Eraser-str committed:
Fixed markdown typo

michaelrkn committed:
Merge pull request #1002 from Eraser-str/patch-1

Fixed markdown typo

michaelrkn committed:
Merge pull request #986 from tomdale/lifecycle-tweaks

Tweaks to component lifecycle guide

hlubek committed:
Update loading-and-error-substates.md

The route names that are used in the example should be consistent.

michaelrkn committed:
Merge pull request #1003 from hlubek/patch-1

Make example with loading substates for nested routes more consistent

michaelrkn committed:
Merge pull request #916 from bmichelsen/add-html-proofer

Add HTML Proofer

tomdale committed:
Implement footer

michaelrkn committed:
update index with info on tutorial

michaelrkn committed:
clean up ember data tutorial

michaelrkn committed:
change prose so getting started is not coupled to tutorial

michaelrkn committed:
remove version number from expected result

michaelrkn committed:
clean up tutorial

Commit:
Fix typo on welcome guide.

michaelrkn committed:
Merge pull request #1004 from daniellutz/patch-1

Fix typo on welcome guide.

Commit:
Update NodeJS installation info.

michaelrkn committed:
Merge pull request #998 from mwils/master

Added spellchecking during build

locks committed:
Merge pull request #1005 from daniellutz/patch-1

Update NodeJS installation info.

alexspeller committed:
Remove resetNamespace from defining routes guide

michaelrkn committed:
Merge pull request #1006 from alexspeller/fix-defining-routes

Remove resetNamespace from defining routes guide

michaelrkn committed:
Add info about handling validation errors

acorncom committed:
Fixed spelling mistake in the readme

michaelrkn committed:
Merge pull request #1010 from acorncom/patch-2

Fixed spelling mistake in the readme

clekstro committed:
Small grammar fix to clarify meaning

locks committed:
Merge pull request #1011 from fauxton/small-grammar-fix

Small grammar fix to clarify meaning

locks committed:
Merge pull request #1009 from emberjs/michaelrkn-patch-1

Add info about handling validation errors

nubs committed:
Remove hyphen from "well separated" after noun.

When a compound adjective comes after the noun, it generally shouldn't be hyphenated: http://www.grammarly.com/handbook/punctuation/hyphen/4/hyphen-with-noun-adjective-or-adverb-and-present-p/.

michaelrkn committed:
Merge pull request #1012 from nubs/patch-1

Remove hyphen from "well separated" after noun.

nubs committed:
Fix a typo with a missing "a."

michaelrkn committed:
Merge pull request #1013 from nubs/patch-2

Fix a typo with a missing "a."

michaelrkn committed:
bring mime-types version down to support ruby 1.9.3

michaelrkn committed:
Remove reference to error.status

acorncom committed:
Fixed some logic issues that seem to have changed

Seems like rspec-core may have had some changes that caused this
to break: https://github.com/rspec/rspec-core/blob/master/Changelog.md

michaelrkn committed:
Merge pull request #1015 from acorncom/fix-for-redesign-test-issues

[Redesign] RSpec test fixes

michaelrkn committed:
update tutorial prose per @acorncom's suggestions

michaelrkn committed:
add info about upgrading ember

michaelrkn committed:
update spelling exceptions

StevenMcD committed:
Corrected minor grammatical error

michaelrkn committed:
Merge pull request #1017 from StevenMcD/patch-1

Corrected minor grammatical error

acorncom committed:
Merge pull request #1014 from emberjs/michaelrkn-patch-1

Remove reference to error.status

michaelrkn committed:
fix problems with normalize not importing

Commit:
Add full lifecycle hook documentation

add some more detail

Fix display

Add didRender detail, fix links

continue to flesh out didRender, add didDestroyElement

add some super calls to the examples, add more details

add example for didUpdateAttrs

Couple of spelling fixes

fix link

address comments from michaelrkn

adhere to new line break guidline

Incorporate some updates based on review comments

remove reference to this.attrs

Remove "Things that trigger a re-render" section in favor of eventual
api doc

remove more this.attrs references

acorncom committed:
Fixed a few minor CSS issues

Our code block text was too small and we had some weird margin
issues on our body tag that was adding spacing around the body
when we didn't want that

acorncom committed:
Merge pull request #650 from emberjs/redesign

Style redesign

acorncom committed:
Merge pull request #1016 from emberjs/upgrading

Add info about upgrading ember

acorncom committed:
Merge pull request #801 from emberjs/tutorial

Tutorial

HeroicEric committed:
Fix indentation

michaelrkn committed:
Merge pull request #1026 from HeroicEric/patch-1

Fix indentation

HeroicEric committed:
Use quotations consistent with rest of guides

michaelrkn committed:
Merge pull request #1027 from HeroicEric/tut-quotes

Use quotations consistent with rest of guides

gpluta committed:
Make mirage paths consistent with the rest

I think it is more clear when we keep pointing the full path  every time. Otherwise things might get a bit confusing.

mitchlloyd committed:
Add documentation for `getOwner`

michaelrkn committed:
Merge pull request #1028 from gpluta/patch-1

Make mirage paths consistent with the rest

Gpx committed:
Fix typo

Gpx committed:
Update example to use ES6 template-strings

michaelrkn committed:
Merge pull request #1031 from Gpx/patch-1

Fix typo

acorncom committed:
Merge pull request #1032 from Gpx/patch-2

Update example to use ES6 template-strings

sircharleswatson committed:
add a <p> tag to the last code sample

This will match the proposed picture in the beginning of the page a little better.

zachfedor committed:
fix typo in applications-and-instances.md

michaelrkn committed:
Merge pull request #1037 from zachfedor/patch-1

fix typo in applications-and-instances.md

michaelrkn committed:
Merge pull request #1035 from sircharleswatson/patch-1

add a <p> tag to the last code sample

kirkland committed:
Fix English

michaelrkn committed:
Merge pull request #1039 from kirkland/patch-1

Fix English

kirkland committed:
Grammar fixes

kirkland committed:
Grammar and style fixes

acorncom committed:
Merge pull request #1040 from kirkland/patch-2

Grammar fixes

acorncom committed:
Merge pull request #1041 from kirkland/patch-3

Grammar and style fixes

Commit:
Removed webkit-font-smoothing to ensure legibility

Since the new Guides now use light on dark code blocks, the `-webkit-font-smoothing` property makes thinner fonts really thin. While that might look good in certain settings it also reduces readability.

Focussed only on the code snippets as requested by @acorncom

gpluta committed:
Tweak the description of the .create() method

The previous description was not in sync with reality. I hope my proposition is more accurate.

michaelrkn committed:
Merge pull request #1043 from gpluta/patch-1

Tweak the description of the .create() method

fzdvincent committed:
Fix incorrect documentation links

The links used in the "Escaping HTML Content" subsection were clobbering the links set in the "Class-based Helpers" subsection.

trek committed:
Merge pull request #1044 from fzdvincent/patch-1

Fix incorrect documentation links

nicholalexander committed:
updated router code snippet to reflect the router being declared a const and not a var by ember

nicholalexander committed:
add viewport

media query and styling on select box to prevent it overlapping header on mobile

use neat variable in sass

acorncom committed:
Merge pull request #1046 from nicholalexander/tutorialFixRoutesSnippet

Fix code snippet showing Router definition

acorncom committed:
Merge pull request #1034 from pichfl/patch-1

Removed webkit-font-smoothing to ensure legibility

nicholalexander committed:
create originals folder and add original images

upgrade the image to the master image

removal of other formats, leaving only svg as reference file

rtablada committed:
Tone for routes and Templates

Routes and templates trickier we v you corrections

Update tone and POV for model-hook

Tone and POV Ember Data

Tone and POV simple component

Updated to PR comments

Ember data better readability for `update config`

Clarify Mirage description

Update model hook to match recommendations

More over punctuation fixed

This comma got away from the blasters

locks committed:
Merge pull request #1049 from rtablada/tone/tutorials

Tutorials - Tone and POV

rtablada committed:
Whitespace/Line Length cleanup Tutorial

rtablada committed:
Addon and application whitespace & line length

michaelrkn committed:
Merge pull request #1051 from rtablada/whitespace/new-line

[WIP] Whitespace/new line clean up

michaelrkn committed:
Update line break guidelines

maxdignan committed:
Update debugging.md

Grammar correction

michaelrkn committed:
Merge pull request #1048 from nicholalexander/images

create original images folder and upgrade ember-core-concepts.png

michaelrkn committed:
Merge pull request #1055 from maxdignan/patch-2

Update debugging.md

michaelrkn committed:
Merge pull request #1052 from nicholalexander/mobileResponsive

meta viewport to fix mobile display

frunjik committed:
[DOC] JSON data for Sideloaded Data is invalid

Its missing a '['

michaelrkn committed:
Merge pull request #1056 from frunjik/patch-1

[DOC] JSON data for Sideloaded Data is invalid

michaelrkn committed:
make font antialiasing consistent in firefox

AlexKVal committed:
Fix typo

It seems the `as` is out of place in this sentence:
`...meaning that Ember will call it for us as during different times in our app`

michaelrkn committed:
Merge pull request #1059 from AlexKVal/patch-1

Fix typo

acorncom committed:
Merge pull request #1057 from emberjs/font

Make font antialiasing consistent in firefox

frunjik committed:
[DOC] JSON definition of Sideloaded Data is wrong

The JSON is now valid,
but the data is wrong ... the ']' needed to be removed instead of added ...

(sorry)

nightire committed:
Fix #1058 Remove font smoothing

Signed-off-by: Albert Yu <dev.yufan@gmail.com>

Commit:
Update model-hook.md

Spelling correction.

rwjblue committed:
Merge pull request #1064 from aaron0browne/patch-1

Update model-hook.md

locks committed:
Merge pull request #1063 from nightire/master

Fix #1058 Remove font smoothing

locks committed:
Merge pull request #1053 from emberjs/michaelrkn-patch-1

Update line break guidelines

michaelrkn committed:
Revert "Fix #1058 Remove font smoothing"

michaelrkn committed:
Merge pull request #1060 from frunjik/patch-2

[DOC] JSON definition of Sideloaded Data is wrong

locks committed:
Merge pull request #1066 from emberjs/revert-1063-master

Revert "Fix #1058 Remove font smoothing"

Commit:
adds more details to transforms section #1061

acorncom committed:
Merge pull request #1069 from skaterdav85/master

adds more details to transforms section #1061

Dhaulagiri committed:
Remove trailing "

rwjblue committed:
Merge pull request #1070 from Dhaulagiri/patch-1

Remove trailing "

Commit:
adds semicolon like other code blocks

locks committed:
Merge pull request #1072 from skaterdav85/patch-1

adds semicolon like other code blocks

Commit:
adds semicolons

acorncom committed:
Merge pull request #1073 from skaterdav85/patch-2

adds semicolons

Commit:
links each mention of a serializer to its corresponding API page

michaelrkn committed:
Merge pull request #1074 from skaterdav85/master

links each mention of a serializer to its corresponding API page

Commit:
adds a section on creating custom transforms

acorncom committed:
Merge pull request #1075 from skaterdav85/master

adds a section on creating custom transforms

Commit:
Renamed Working with Relationships to just Relationships and moved it after Creating, Updating and Deleting. Also removes some stuff about relationships in Creating, Updating and Deleting and put in Relationships. #1076

removes defining-relationships file and puts contents in relationships file

removes the h2 because an h3 followed directly by an h2 causes a build issue which results in a mispelling

acorncom committed:
Merge pull request #1077 from skaterdav85/master

#1076 break out "Defining Relationships" section

Commit:
replaces `var store` with `this.store` to be consistent with other store references

acorncom committed:
Merge pull request #1079 from skaterdav85/master

var store updates

locks committed:
Replaces `var` with `let`

michaelrkn committed:
Merge pull request #1080 from emberjs/locks-patch-1

Replaces `var` with `let`

Commit:
minor formatting

michaelrkn committed:
Merge pull request #1081 from skaterdav85/master

minor formatting

locks committed:
Replaces this.attrs with this.get

thijsvdanker committed:
Inject stub service as proper property name

The component uses the `locationService` to inject the location service, while the test injects it as `location`.

acorncom committed:
Merge pull request #1082 from emberjs/locks-patch-1

Replaces this.attrs with this.get

michaelrkn committed:
use cleaner approach to make support-for available

locks committed:
Merge pull request #1085 from emberjs/support-for

Use cleaner approach to make support-for available

homu committed:
Auto merge of #1083 - thijsvdanker:patch-1, r=locks

Inject stub service as proper property name

The component uses the `locationService` to inject the location service, while the test injects it as `location`.

Commit:
Fix examples by updating references to new variable name.  Added comment
on inject.

update comment with punctuation

homu committed:
Auto merge of #1086 - toddjordan:fix-example-breakage, r=locks

Fix the last change that broke example

Fix examples by updating references to new variable name.  Added comment on inject to explain where its being injected.

steveszc committed:
Create TROUBLESHOOTING.md

Includes solution for failed bundle on OS X El Capitan

acorncom committed:
Merge pull request #1087 from steveszc/add-troubleshooting

Create TROUBLESHOOTING.md

acorncom committed:
Troubleshooting file cleanup

steveszc committed:
Create .editorconfig

basic editorconfig for middleman

steveszc committed:
Enhanced nav and ToC on small screens

-Added wai-aria roles and labels as appropriate
- Added Collapsable ToC on small screens
- Added condensed nav menu on small screens

rtablada committed:
Fixes syntax highlighting for index template

michaelrkn committed:
Merge pull request #1091 from rtablada/fix/index-template-highlighting

Fixes syntax highlighting for index template

rtablada committed:
Instructions to reboot serve after addon install

locks committed:
Adds paragraph tag to make it more consistent

rtablada committed:
Remove truncated templates from component tut

michaelrkn committed:
Merge pull request #1096 from rtablada/fix/non-trucated-file-examples

Fix non-trucated file examples

michaelrkn committed:
Merge pull request #1095 from emberjs/locks-patch-1

Adds paragraph tag to make it more consistent

michaelrkn committed:
Merge pull request #1097 from rtablada/mirage-server-restart

Instructions to reboot serve after addon install

michaelrkn committed:
Merge pull request #1088 from steveszc/EditorConfig

Create .editorconfig

locks committed:
Add maintainers section to README

rtablada committed:
Fix missing link-to on index.hbs

michaelrkn committed:
consolidate troubleshooting tips

homu committed:
Auto merge of #1103 - emberjs:trouble, r=locks

Consolidate troubleshooting tips

rwjblue committed:
Merge pull request #1029 from mitchlloyd/get-owner-lookups

Add documentation for `getOwner`

rwjblue committed:
Revert "Add documentation for `getOwner`"

homu committed:
Auto merge of #1108 - emberjs:revert-1029-get-owner-lookups, r=rwjblue

Revert "Add documentation for `getOwner`"

Reverts emberjs/guides#1029

prastut committed:
Update routes-and-templates.md

The exact URL isn't specified when we first try to test the ember application. Since we haven't defined the "index" route, nothing would be displayed to the user at that particular time.
It took me a while to realize, so I guess the link should be edited.

homu committed:
Auto merge of #1109 - prastut:patch-1, r=locks

Update routes-and-templates.md

The exact URL isn't specified when we first try to test the ember application. Since we haven't defined the "index" route, nothing would be displayed to the user at that particular time.
It took me a while to realize, so I guess the link should be edited.

locks committed:
Clarify pull request process a smidge

michaelrkn committed:
Merge pull request #1113 from emberjs/locks-patch-1

Clarify pull request process a smidge

thijsvdanker committed:
Update transitionTo and transitionToRoute links

The links didn't point to the corresponding functions in the API.

homu committed:
Auto merge of #1115 - thijsvdanker:patch-2, r=rwjblue

Update transitionTo and transitionToRoute links

The links didn't point to the corresponding functions in the API.

sevenfour committed:
Update simple-component.md

Fixed minor typo at the last sentence: toggle images instead of toggle or images

homu committed:
Auto merge of #1116 - sevenfour:patch-1, r=locks

Update simple-component.md

Fixed minor typo at the last sentence: toggle images instead of toggle or images

mozeryansky committed:
Changed inline code to shell code

Trying to be consistent. Also

homu committed:
Auto merge of #1121 - mozeryansky:patch-1, r=locks

Changed inline code to shell code

Trying to be consistent. Also

mozeryansky committed:
Fix JSHint Error

mirage/config.js: line 38, col 6, Missing semicolon.

homu committed:
Auto merge of #1130 - mozeryansky:patch-2, r=locks

Fix JSHint Error

mirage/config.js: line 38, col 6, Missing semicolon.

mozeryansky committed:
Seperate execution lines from shell output

To maintain consistency in the guides

mozeryansky committed:
Fixed ember template error

With the '/' character immediately at the end, reports an error:
`An unquoted attribute value must be a string or a mustache, preceeded by whitespace or a '=' character, and followed by whitespace or a '>' character (on line 8)`

Adding a space fixes the error.

mozeryansky committed:
Fixed ember template error

Same as other commit

mozeryansky committed:
Example for uncountable('advice')

homu committed:
Auto merge of #1135 - mozeryansky:patch-6, r=locks

Example for uncountable('advice')

sbeleidy committed:
Correct punctuation in application route

rwjblue committed:
Merge pull request #1136 from sbeleidy/patch-2

Correct punctuation in application route

homu committed:
Auto merge of #1134 - mozeryansky:patch-5, r=locks

Fixed ember template error

With the '/' character immediately at the end, reports an error:
`An unquoted attribute value must be a string or a mustache, preceeded by whitespace or a '=' character, and followed by whitespace or a '>' character (on line 8)`

Adding a space fixes the error.

izzygomez committed:
update routes-and-templates.md

homu committed:
Auto merge of #1144 - izzygomez:patch-1, r=locks

update routes-and-templates.md

acorncom committed:
Merge pull request #1089 from toddjordan/document-lifecycle-hooks

Add lifecycle hook documentation

homu committed:
Auto merge of #1132 - mozeryansky:patch-2, r=acorncom

Seperate execution lines from shell output

To maintain consistency in the guides

jsmestad committed:
Remove whitespace from conditional

jsmestad committed:
Fix other occurrences

homu committed:
Auto merge of #1151 - jsmestad:patch-1, r=locks

Remove whitespace from conditional on components guide

acorncom committed:
Tweaking ember-cli install instructions so we don't lose folks

Znegl committed:
Update customizing-a-components-element.md

Added information about how to remove unused bound attributes from the DOM.

Znegl committed:
Update customizing-a-components-element.md

Fixed typo

homu committed:
Auto merge of #1153 - acorncom:tweaking-ember-cli-install-instructions, r=locks

Tweaking ember-cli install instructions so we don't lose folks

Fixes #1094

@locks, here's the PR for the changes we discussed earlier today

jsmestad committed:
remove px from width attribute

homu committed:
Auto merge of #1156 - jsmestad:fix-pixels-in-width, r=locks

remove px from width attribute in tutorial

This is a redo of PR #1152

Znegl committed:
Update customizing-a-components-element.md

Clarified information about attribute rendering.

locks committed:
Update feature-flags.md

homu committed:
Auto merge of #1160 - emberjs:locks-patch-3, r=locks

Update feature-flags.md

acorncom committed:
Updating version number

homu committed:
Auto merge of #1154 - Znegl:patch-1, r=locks

Update customizing-a-components-element.md

Added information about how to remove unused bound attributes from the DOM.

homu committed:
Auto merge of #1165 - emberjs:acorncom-patch-1, r=locks

Updating version number
